flixel/FlxSprite.hx

```diff
@@ -275,7 +275,14 @@ class FlxSprite extends FlxObject
 	 * Set to `null` to discard graphic frame clipping.
 	 */
 	public var clipRect(default, set):FlxRect;
-
+	
+	/**
+	 * If true, clipRect will behave as it did in flixel 4 and earlier.
+	 * Where the clipRect will scale up proportionally with the sprite.
+	 * @since 5.0.0
+	 */
+	public var clipRectIgnoreScale(default, set):Bool = false;
```

Comment:
TODO:
- keep legacy behavior, make this an opt-in feature
- come up with a better name maybe `scaleClipRect` or something